pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -280,7 +289,7 @@ func (c *MizarPodController) handle(keyWithEventType KeyWithEventType) error {
 				klog.Errorf("Pod name (%s) - update pod's annotation to API server in error (%v) when (%v).", obj.Name, err, eventType)
 				return err
 			}
-			klog.Infof("Pod name (%s) - update pod's annotation to API server successfully when (%v).", obj.Name, eventType)
+			klog.V(2).Infof("Pod name (%s) - update pod's annotation to API server successfully when (%v).", obj.Name, eventType)
```

Comment:
nit: is V(4) better (less verbose) then V(2)?

Reply:
will update in later changes.